test/uk/gov/hmrc/apiplatformjobs/repository/UnusedApplicationsRepositorySpec.scala

```diff
@@ -125,6 +125,7 @@ class UnusedApplicationsRepositorySpec
           )
           .toFuture()
       )
+      println("XXXX " + await(unusedApplicationRepository.collection.countDocuments().toFuture()))
```

Comment:
I guess this was for debugging only?

Reply:
No - it's what makes the whole thing work :-)

Reply:
As are the other two printlns... :-O